src/cci/cci_query_execute.c

```diff
@@ -3613,6 +3613,7 @@ int
 qe_get_data_int (T_CCI_U_TYPE u_type, char *col_value_p, void *value)
 {
   int data;
+  int *temp_value = (int *) value;
```

Comment:
**temp_value** is also ok. However, the **i_value** I commented on is the use of '**i**' as the meaning of the (**int ***) type of the pointer **_value_**. But its trivial. Do as you like.